Tools/survey_helper.py

```diff
@@ -0,0 +1,67 @@
+import json
+from Constants.prompts import CHAT_MAIN_PROMPT, SURVEY_QUESTION_PROMPT
+from Tools.openai_tools import OpenAIClient
+from SqlDB.user_details_service import save_user_details
+from config import Config
+
```

Comment:
This whole module can be wrapped into the OpenAiUserDetailsManager or something similar.